filewatcher/filewatcher_test.go

```diff
@@ -255,9 +245,6 @@ func TestParserFailure(t *testing.T) {
 	}
 	// Give it some time to handle the file content change
 	time.Sleep(interval * 10)
-	if atomic.LoadInt64(&loggerCalled) == 0 {
```

Comment:
I'm not sure how we can test the logger call without injecting the logger. Is it ok to remove the check or could you recommend a way to test it without injection?